ZipZipSa/ZipZipSa/Data/Model/PlaceRespnse.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Untitled.swift
+//  ZipZipSa
+//
+//  Created by 조우현 on 11/23/24.
+//
+
+import Foundation
+
+struct PlacesResponse: Codable {
+    let results: [PlaceResult]
+}
+
+struct PlaceResult: Codable {
+    let geometry: Geometry
+}
+
+struct Geometry: Codable {
+    let location: Location
+}
+
+struct Location: Codable {
+    let lat: Double
+    let lng: Double
+}
```

Comment:
S1
전의 애공 코드를 빌려오자면, Codable한 경우  enum CodingKeys: String, CodingKey를 사용할 수 있답니다!
이건 서버에서 보내주는 키 값을 맞춰주기 위해 쓰는건데요, 서버에서 보내주는 값을 rawValue로 넣어주면 클라단에서는 우리가 원하는 변수 명을 쓰면서 서버 키 값을 맞춰주기까지 동시에 된답니당!

```suggestion
struct Location: Codable {
     let latitude: Double
     let longitude: Double

    enum CodingKeys: String, CodingKey {
        case latitude = "lat"
        case longitude = "lng"
    }
}
```

Reply:
오오 코딩키가 이럴때 쓰는거균뇨! 또 하나 배워갑니다 👍